sdk/formrecognizer/ai-form-recognizer/src/formRecognizerClient.ts

```diff
@@ -144,17 +177,22 @@ export interface BeginRecognizeReceiptsOptions extends BeginRecognizeFormsOption
   locale?: string;
 }
 
+/**
+ * Options for starting the receipt recognition operation
+ */
+export type BeginRecognizeReceiptsOptions = BeginRecognizePrebuiltOptions;
+
 /**
  * Options for starting the Business Card recognition operation
  */
-export interface BeginRecognizeBusinessCardsOptions extends BeginRecognizeFormsOptions {
-  /**
-   * Locale of the document.
-   *
-   * Supported locales include: en-AU, en-CA, en-GB, en-IN, en-US (default if none provided).
-   */
-  locale?: string;
-}
+export type BeginRecognizeBusinessCardsOptions = BeginRecognizePrebuiltOptions;
+
+/**
+ * Options for starting the Invoice recognition operation
+ */
+export type BeginRecognizeInvoicesOptions = BeginRecognizePrebuiltOptions;
```

Comment:
Starting out Invoice will only support en-US. Not sure if that changes this code for you.